v2_1/ws/rest/src/SDMXRestTypes.xsd

```diff
@@ -29,7 +29,7 @@
         </xs:annotation>
         <xs:restriction base="xs:string">
             <!-- (common:NestedNCNameIDType\,)?(common:IDType) -->
-            <xs:pattern value="([A-Za-z][A-Za-z0-9_\-]*(\.[A-Za-z][A-Za-z0-9_\-]*)*[,])?([A-Za-z0-9_@$\-]+)"/>
+            <xs:pattern value="([A-Za-z][A-Za-z0-9_\-]*(\.[A-Za-z][A-Za-z0-9_\-]*)*[,])?([A-Za-z0-9_@$\-]+)([+]([A-Za-z][A-Za-z0-9_\-]*(\.[A-Za-z][A-Za-z0-9_\-]*)*[,])?([A-Za-z0-9_@$\-]+))*"/>
```

Comment:
The expression ```ECB.DISS+ESTAT``` should not be acceptable, since a single token (without commas) should correspond to the Provider ID, hence no ```.``` is allowed.

Reply:
You are right. 
This is what is accepted instead: `ECB.DISS,MFS+ESTAT` meaning:
- `ECB.DISS,MFS` or 
- `ESTAT` 

may provide data

Reply:
True, but in that case ```ESTAT``` would be interpreted as a Provider ID, not an Agency ID.

Reply:
i.e.: `all,ESTAT`